controllers/imagejob/imagejob_controller.go

```diff
@@ -133,6 +147,16 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 			Image:           givenImage.Image,
 			Name:            givenImage.Name,
 			ImagePullPolicy: givenImage.ImagePullPolicy,
+			Resources: v1.ResourceRequirements{
+				Requests: v1.ResourceList{
+					"cpu":    resource.MustParse("10"),
```

Comment:
please adjust these accordingly

Reply:
ah looks like you have it on the other pr, can you make sure to only have it over there then and not in this pr